src/touch.js

```diff
@@ -0,0 +1,89 @@
+var common = require('./common');
+var fs = require('fs');
+
+module.exports = _touch;
```

Comment:
It's against my style, but I favor consistency so if that's the case then I suppose I can change it.